config/crd/bases/ndb.nutanix.com_ndbservers.yaml

```diff
@@ -35,13 +45,63 @@ spec:
           spec:
             description: NDBServerSpec defines the desired state of NDBServer
             properties:
-              foo:
-                description: Foo is an example field of NDBServer. Edit ndbserver_types.go
-                  to remove/update
+              credentialSecret:
                 type: string
+              server:
```

Comment:
would this require changes in the helm repo too?

Reply:
Yes this will require us to add NDB CRD to the helm charts too. We should do that once the DB controller is refactored to make use of the NDB CR. WDYT?